framework/src/base/MeshGeneratorSystem.C

```diff
@@ -347,6 +347,16 @@ MeshGeneratorSystem::executeMeshGenerators()
       const auto & name = generator->name();
 
       auto current_mesh = generator->generateInternal();
+#ifdef DEBUG
```

Comment:
```suggestion
#ifndef NDEBUG
```

?

Reply:
this check adds asymptotic slowness. Conventionally in that case we do not add it to `devel`

Reply:
okay